package.json

```diff
@@ -40,15 +40,15 @@
     "node": ">=8.0.0"
   },
   "devDependencies": {
-    "@babel/cli": "^7.5.0",
-    "@babel/core": "^7.5.0",
-    "@babel/preset-env": "^7.5.0",
+    "@babel/cli": "^7.8.4",
+    "@babel/core": "^7.9.6",
+    "@babel/preset-env": "^7.9.6",
     "@metarhia/doc": "^0.6.0",
-    "eslint": "^6.0.1",
+    "eslint": "^7.1.0",
```

Comment:
Same here, even though it's just a dev dependency, `eslint` v7 doesn't support Node.js 8.